docs/development/build-instructions-gn.md

```diff
@@ -78,6 +78,7 @@ origin URLs.
 $ cd src/electron
 $ git remote remove origin
 $ git remote add origin https://github.com/electron/electron
+$ git fetch
```

Comment:
Can you further explain why we need `git fetch` here.  The raw content is identical and refetching from the remote vs using the existing git objects that came from the local cache should be a no-op (but a costly no-op)

Reply:
Sure, as I follow the instructions here, I encountered an error
```
❯ git branch --set-upstream-to=origin/main
fatal: the requested upstream branch 'origin/main' does not exist
```
So I think we need to fetch the remote here.